codec_specs.md

```diff
@@ -559,36 +559,34 @@ Initialization: none
 
 Codec ID: A_AC3
 
-Codec Name: Dolby Digital AC-3
+Codec Name: Dolby Digital / AC-3
 
-Description: BSID <= 8 !! The private data is void ??? Corresponding ACM wFormatTag : 0x2000 ; channel number have
-to be read from the corresponding audio element
+Description: Individual frames of AC-3 `syncframe()` stored as described in [@!ATSC-A52] or [@!ETSI-TS102-366] when the value of the `bsid` field defined in Section 5.4.2.1 of [@!ATSC-A52] or Section 4.4.2.1 of [@!ETSI-TS102-366] is 10 or below.
+Corresponding ACM wFormatTag : 0x2000 ; channel number have to be read from the corresponding audio element
 
 ### A_AC3/BSID9
 
 Codec ID: A_AC3/BSID9
 
-Codec Name: Dolby Digital AC-3
+Codec Name: Dolby Digital / AC-3
 
-Description: The ac3 frame header has, similar to the mpeg-audio header a version field. Normal ac3 is defined as bitstream id 8 (5 Bits, numbers are 0-15).
-Everything below 8 is still compatible with all decoders that handle 8 correctly.
-Everything higher are additions that break decoder compatibility.
-For the samplerates 24kHz (00); 22,05kHz (01) and 16kHz (10) the BSID is 9
-For the samplerates 12kHz (00); 11,025kHz (01) and 8kHz (10) the BSID is 10
+Description: Individual frames of AC-3 `syncframe()` stored as described in [@!ATSC-A52] or [@!ETSI-TS102-366] when the value of the `bsid` field defined in Section 5.4.2.1 of [@!ATSC-A52] or Section 4.4.2.1 of [@!ETSI-TS102-366] is 9.
+Note that the value 9 in the `bsid` field is not standard but it is defacto used for dividing the sampling rate defined in Section 5.4.1.3 of [@!ATSC-A52] or Section 4.4.2.1 of [@!ETSI-TS102-366] by 2.
+
+Using this Codec ID is **NOT RECOMMENDED** as many Matroska Players don't support it whereas they support `A_AC3` with a `bsid` of 9.
```

Comment:
I'd be even clearer in the recommendation, maybe something like

> Using this Codec ID is **NOT RECOMMENDED** as many Matroska players don't support it. The generic Codec ID `A_AC3` should be used instead.

Reply:
OK, although I'd keep the fact that this bsid value is actually supported by such players.

Reply:
That's fine. I think I only want to make it clear for implementers of writers that they should just use `A_AC3`, and for implementers of readers that they can usually handle `A_AC3` & `A_AC3/BSID<n>` identically.